modules/collabora_online_group/src/Plugin/Group/RelationHandler/CollaboraPermissionProvider.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\collabora_online_group\Plugin\Group\RelationHandler;
+
+use Drupal\groupmedia\Plugin\Group\RelationHandler\GroupMediaPermissionProvider;
+
+/**
+ * Provides Collabora permissions for group.
+ */
+class CollaboraPermissionProvider extends GroupMediaPermissionProvider {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildPermissions(): array {
+    $permissions = $this->parent->buildPermissions();
+
+    /** @see \Drupal\group\Plugin\Group\RelationHandlerDefault\PermissionProvider::buildPermissions() */
+    $provider_chain = $this->groupRelationTypeManager()->getPermissionProvider($this->pluginId);
+
+    // Add Collabora permissions.
+    $prefix = 'Entity:';
```

Comment:
The `$prefix` variable is weird, could just be hard-coded, but ok for me to align with other implementations in group module.

Reply:
Indeed, wanted to stay aligned with other implementations.